### PR TITLE
Issue #90 - Test resource inclusion

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: tycho-compiler-plugin/src/test/java/org/eclipse/tycho/osgicompiler/test/OsgiCompilerTest.java [hunk 1]
```diff
@@ -210,6 +210,15 @@ private void assertSameFile(File expected, File actual) throws IOException {
         assertEquals(expected.getCanonicalFile(), actual.getCanonicalFile());
     }
 
+    public void testCopyResourcesNoConf() throws Exception {
+        File basedir = getBasedir("projects/resources/p004");
+        List<MavenProject> projects = getSortedProjects(basedir, null);
+        MavenProject project = projects.get(0);
+        getMojo(projects, project).execute();
+        assertTrue(new File(project.getBasedir(), "target/classes/testresources/Test.class").canRead());
+        assertTrue(new File(project.getBasedir(), "target/classes/testresources/test.properties").canRead());
```
Comment (laeubi) [inline tycho-compiler-plugin/src/test/java/org/eclipse/tycho/osgicompiler/test/OsgiCompilerTest.java:219]: as this file is not mentioned in bin.includes should it really be copied?

Comment (mickaelistria) [reply tycho-compiler-plugin/src/test/java/org/eclipse/tycho/osgicompiler/test/OsgiCompilerTest.java:219]: Yes, the expected behavior is that the resources that are part of source folders are copied into the same folder as .class files.

Comment (laeubi) [reply tycho-compiler-plugin/src/test/java/org/eclipse/tycho/osgicompiler/test/OsgiCompilerTest.java:219]: You right I was confused by the path :-\

**Comment by mickaelistria** [comment]
As this is an important regression, I'm merging this test. It's fine to get all builds failing until this is resolved, as fixing this is highest priority and worth blocking everything else.

**Pull request merged**